offchain/hello-world-api/src/Api.purs

```diff
@@ -124,14 +124,18 @@ redeemFromScript
   -> Contract () Unit
 redeemFromScript vhash validator txInput = do
   utxos <- getUtxos (scriptHashAddress vhash)
+  key <- liftContractM "no wallet" =<< ownPaymentPubKeyHash
   let
     lookups :: Lookups.ScriptLookups PlutusData
     lookups = Lookups.validator validator
       <> Lookups.unspentOutputs utxos
 
     constraints :: TxConstraints Unit Unit
     constraints = Constraints.mustSpendScriptOutput txInput spendRedeemer
-  _ <- buildBalanceSignAndSubmitTx lookups constraints
+      <> Constraints.mustBeSignedBy key
```

Comment:
Could we add a todo here to remove this workaround once CTL/Nami resolves https://github.com/Plutonomicon/cardano-transaction-lib/issues/1079?

Reply:
:+1: @leomayleomay I pushed this Todo cause I should've added it in the first place and it shouldn't conflict you.

Reply:
:+1: